### ISSUE TITLE
DB data source

### ISSUE THREAD

**Issue by koolaidjones** [revanthb3000/FIFA-Analysis#1]
Are you scraping Futhead? If so I can send you the actual db data from EA in excel if you would like.

**Comment by revanthb3000** [comment]
Yes, I am using FutHead as my source (and have already made a DB). Actual DB data from EA ? What do you mean ? Is that extracted from the game's data files ?
